bitnami/rabbitmq/values.yaml

```diff
@@ -1434,17 +1540,17 @@ metrics:
 volumePermissions:
   ## @param volumePermissions.enabled Enable init container that changes the owner and group of the persistent volume(s) mountpoint to `runAsUser:fsGroup`
   ##
-  enabled: false
-  ## @param volumePermissions.image.registry [default: REGISTRY_NAME] Init container volume-permissions image registry
-  ## @param volumePermissions.image.repository [default: REPOSITORY_NAME/os-shell] Init container volume-permissions image repository
-  ## @skip volumePermissions.image.tag Init container volume-permissions image tag
+  enabled: true
+  ## @param volumePermissions.image.registry Init container volume-permissions image registry
+  ## @param volumePermissions.image.repository Init container volume-permissions image repository
+  ## @param volumePermissions.image.tag Init container volume-permissions image tag
```

Comment:
I'm totally ok with the false -> true (it's an old patch).
But you should keep untouched the comments (=> keep the recent comments from bitnami, they are more up-to-date, and it limits the size of our patchs so reduce conflicts in the future)